hack/lib/init.sh

```diff
@@ -46,4 +46,9 @@ unset library_files library_file init_source
 
 # all of our Bash scripts need to have the stacktrace
 # handler installed to deal with errors
-os::log::stacktrace::install
\ No newline at end of file
+os::log::stacktrace::install
+
+# all of our Bash scripts need to have acces to the
```

Comment:
s/acces/access/.

While I'm at it, s/all/All/, and the sentence needs a period at the end.